tokio-quiche/src/lib.rs

```diff
@@ -194,9 +194,7 @@ where
             #[cfg_attr(not(target_os = "linux"), expect(unused_mut))]
             let mut socket = s.try_into()?;
             #[cfg(target_os = "linux")]
-            socket.apply_max_capabilities(
-                params.settings.max_send_udp_payload_size,
```

Comment:
params argument seems unused.  Could it be removed from listen()?

Reply:
`params` is piped down into `listen_with_capabilities`

Reply:
You may need to rename to _params

Reply:
Piping it to `listen_with_capabilities` is still considered as using the parameter

Reply:
I missed that line, sorry :)